concepticondata/conceptlists/Williamson-1973-118.tsv

```diff
@@ -0,0 +1,119 @@
+Williamson-1973-118-68	68	press: squeeze out: milk	presser::traire	drücken, pressen: auspressen, ausdrücken:: melken		
+Williamson-1973-118-69	69	quarrel v. and n.	se quereller, disputer: querelle	(sich) streiten: Streit		
+Williamson-1973-118-70	70	raphia palm (vinifera): (hookeri):: date palm	raphia (vinifera): hookeri:: dattier	?: ?:: Dattelpalme		
+Williamson-1973-118-71	71	refuse: retain	refuser: retenir	verweigern, abschlagen: zurückhalten, einbehalten		
+Williamson-1973-118-72	72	resemble: be equal to	ressembler: être égal à	ähnlich sein: gleichen, gleich sein	1777	RESEMBLE
+Williamson-1973-118-73	73	road: path	route: sentier	(Land)Strasse: Pfad, Weg	2457	PATH OR ROAD
+Williamson-1973-118-74	74	roast on coals	griller au feu de bois	auf Kohlen rösten		
+Williamson-1973-118-75	75	rot	pourrir	verfaulen, verwesen	2907	ROT
+Williamson-1973-118-76	76	rubbish-heap	tas d’ordures	Schutthaufen		
+Williamson-1973-118-77	77	send: work n.	envoyer: travail	schicken, senden: Arbeit:: arbeiten		
+Williamson-1973-118-78	78	set a trap	poser un piège	eine Falle stellen		
+Williamson-1973-118-79	79	sew	coudre	nähen	1457	SEW
+Williamson-1973-118-80	80	shine: be light: be white	briller: être pâle:: être blanc	scheinen, leuchten: hell sein, licht sein:: weiss sein		
+Williamson-1973-118-81	81	sing	chanter	singen	1261	SING
+Williamson-1973-118-82	82	skin: hide:: bark	peau:: écorce	Haut: Fell, Haut grösseren Tieren:: (Baum) Rinde, Borke	2127	BARK OR SKIN
+Williamson-1973-118-83	83	smell v.	sentir	riechen	1586	SMELL (PERCEIVE)
+Williamson-1973-118-84	84	snake	serpent	Schlange	730	SNAKE
+Williamson-1973-118-85	85	song	chanson	Lied, Gesang	1096	SONG
+Williamson-1973-118-86	86	speak	parler	sprechen, reden	1623	SPEAK
+Williamson-1973-118-87	87	spit	cracher	spucken	1440	SPIT
+Williamson-1973-118-88	88	spread in sun to dry	secher au soleil	im Sonnenlicht (in der Sonne?) ausbreiten um zu trocknen		
+Williamson-1973-118-89	89	sticky, be	poisser	klebrig sein	3703	STICKY
+Williamson-1973-118-90	90	stone: rock	pierre: rocher	Stein: Fels (Felsen? Felsgestein?)	2125	STONE OR ROCK
+Williamson-1973-118-91	91	take out of: empty	retirer: vider	herausnehmen: leeren		
+Williamson-1973-118-92	92	thing	chose	Ding	738	THING
+Williamson-1973-118-93	93	tongue: language	langue: langue (langage)	Zunge: Sprache		
+Williamson-1973-118-94	94	tooth	dent	Zahn	1380	TOOTH
+Williamson-1973-118-95	95	tortoise	tortue	Schildkröte	895	TORTOISE
+Williamson-1973-118-96	96	tree	arbre	Baum	906	TREE
+Williamson-1973-118-97	97	twin	jumeau (x)	Zwilling	1005	TWINS
+Williamson-1973-118-98	98	want: lack:: seek	vouloir: manquer de:: chercher	wünschen, wollen: nicht haben, Mangel haben an:: suchen		
+Williamson-1973-118-99	99	war	guerre	Krieg	935	WAR
+Williamson-1973-118-100	100	water	eau	Wasser	948	WATER
+Williamson-1973-118-101	101	wing	aile	Flügel	1257	WING
+Williamson-1973-118-102	102	woman: wife	femme: épouse	Weib, Frau: Ehefrau, Gattin		
```

Comment:
Since we have `HUSBAND OR MALE PERSON` we should also have, I think, `WIFE OR FEMALE PERSON`.

Reply:
Might be a candidate for a new concept set, but while mapping I only found 2800 FEMALE PERSON OR MOTHER.